src/Discord.Net/Entities/Webhooks/IWebhook.cs

```diff
@@ -0,0 +1,25 @@
+// todo: docs
+using System.Threading.Tasks;
+
+namespace Discord
+{
+    public interface IWebhook
+    {
+        ValueTask<IUser> GetCreatorAsync(StateBehavior stateBehavior = default, RequestOptions? options = default);
+        ValueTask<ITextChannel> GetChannelAsync(StateBehavior stateBehavior = default, RequestOptions? options = default);
+        ValueTask<IGuild> GetGuildAsync(StateBehavior stateBehavior = default, RequestOptions? options = default);
+        // review: does this need to be an async retrieval? the only path to get an IWebhook should be via an ITextChannel
+
```

Comment:
Doesn't hurt to be consistent. The performance impact should be negligible, particularly when using `ValueTask`.